test/empirical.jl

```diff
@@ -13,3 +13,19 @@ using Test
     @test fnecdf([zeros(5000); ones(5000)]) == [zeros(5000); ones(5000)]
     @test extrema(fnecdf) == (minimum(fnecdf), maximum(fnecdf)) == (0.5, 0.5)
 end
+
+@testset "Weighted ECDF" begin
+    x = randn(10000000)
+    w = weights(rand(10000000))
+    fnecdf = ecdf(x, w)
+    y = [-1.96, -1.644854, -1.281552, -0.6744898, 0, 0.6744898, 1.281552, 1.644854, 1.96]
+    @test fnecdf.weights.values != w.values
+    @test fnecdf(y) ≈ map(fnecdf, y)
```

Comment:
Right. I initially left it out because I wasn't sure about, but I think it's the same test. As in, since the weights are uniform and independent of the sample points, the empirical quantiles should be the same as an evenly weighted sample.